packages/ui/src/constants/zIndex.ts

```diff
@@ -0,0 +1,8 @@
+// Those values are used in each corresponding component
+// Those component needs z-index as the portal is attached to the children container to keep accessibility working properly.
+// Only tooltip is attached to document.body and do not need z-index
+export const Z_INDEX = {
+  menu: 1,
+  popover: 1,
+  modal: 1,
```

Comment:
As Modal and Popover (and Menu) are in the same z-index, wont' it be problematic if you have, say, a popover into a modal?

Reply:
@ModuloM They are made the way that the portal created in order in the children. They all are same zindex so first in dom is shown first. You want to try you can check in Modal story and Popover story there is examples at the bottom